### PR TITLE
proper switch from http to https

### PR DESCRIPTION
**Pull request by jmorel**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var gulp = require('gulp');
-var http = require('http');
-var fs = require('fs');
+var https = require('https');
```
Comment (richterb) [inline gulpfile.js]: Previously, as far as I can remember, the code used to work both for HTTP (e.g. in localhost) and HTTPS portals (production). Was it not the case anymore? Does it work with HTTP with that code, if we want to work/debug locally?

Comment (jmorel) [reply gulpfile.js]: Actually https never worked since the http module doesn't support https (hence the issue and the need for this PR)

Comment (jmorel) [reply gulpfile.js]: And no, it won't work locally. Do we really need it to? If so I can update the client, it's not too big a feature.

Comment (richterb) [reply gulpfile.js]: Not really, it would only be useful for ODS engineers anyway.

**Pull request merged**